composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/Timeslot.kt

```diff
@@ -0,0 +1,161 @@
+                        .fillMaxHeight()
+                        .height(82.dp)
+                        .width(56.31.dp),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    verticalArrangement = Arrangement.Center
+                ) {
+                    Text(text = dateDay, style = dateDayTextStyle)
+                    Text(text = dateMonth, style = dateMonthTextStyle)
+                }
+
+                Column(
+                    Modifier.padding(start = 12.dp, top = 10.dp),
+                ) {
+                    Row {
+                        Text(
+                            text = stringResource(R.string.laadpaal, chargeHubName),
+                            style = cardTitleStyle
+                        )
+                        Spacer(modifier = Modifier.weight(1F))
+                        Icon(
+                            Icons.AutoMirrored.Filled.ArrowForwardIos,
+                            null,
+                            Modifier.padding(bottom = 11.dp, end = 6.dp).clickable { onArrowIconClicked() }
+                        )
+                    }
+
+                    Column {
+                        Row {
+                            Column {
+                                Text(
+                                    text = stringResource(R.string.jouw_tijdslot),
+                                    style = headerTextStyle
+                                )
+                                Text(
+                                    text = stringResource(
+                                        R.string.timeslot_duration_start_end,
+                                        start,
+                                        end
+                                    ), style = textTextStyle
+                                )
+                            }
+
+                            Spacer(Modifier.weight(1F))
+
+                            VerticalDivider(
+                                modifier = Modifier.height(14.dp),
+                                color = Color(0XFF888888)
+                            )
+
+                            Spacer(Modifier.weight(1F))
+
+                            Column {
+                                Text(
+                                    modifier = Modifier.padding(end = 44.69.dp),
+                                    text = stringResource(R.string.oplaadtijd),
+                                    style = headerTextStyle
+                                )
+                                Text(
+                                    modifier = Modifier.padding(end = 44.69.dp),
+                                    text = stringResource(R.string.uur, duration),
+                                    style = textTextStyle
+                                )
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+private val textTextStyle = TextStyle(
```

Comment:
Why are these added here? Weren't these already created? These should not belong here

Reply:
I added them there because that object isn't on develop yet, once that is in (Textfields PR), I will move add them there

Reply:
Maybe add a TODO for this